client/src/containers/HomePage.tsx

```diff
@@ -261,20 +205,6 @@ const HomePage: React.FC<HomePageProps> = ({ useNewPortfolioMethod }) => {
                         A&amp;E Real Estate
                       </Link>
                     </h6>
-                    <Link
```

Comment:
is there a reason why we're removing the logos? it was a nice visual break on our landing page imo

Reply:
after cutting one, they looked a bit unbalanced and leslie decided to shorten the description and drop the images. I think after portfolio pruning we should revisit these and add back logos and a third example. https://app.shortcut.com/justfixnyc/story/12695/nice-to-have-make-example-portfolios-more-visually-consistent-by-removing-logos-and-shortening-stellar-description